google-oauth-client-jetty/src/main/java/com/google/api/client/extensions/jetty/auth/oauth2/LocalServerReceiver.java

```diff
@@ -45,6 +45,8 @@
  */
 public final class LocalServerReceiver implements VerificationCodeReceiver {
 
+  private static final String LOCALHOST = "127.0.0.1";
```

Comment:
I see [the referenced doc](https://developers.google.com/identity/protocols/OAuth2InstalledApp), but it also says:

>  You should query your platform for the relevant loopback IP, and start an HTTP listener on a random available port

In Java, that would be done via `InetAddress.getLoopbackAddress()`. Although I also notice that other parts of the code will break with IPv6. Although in the same document it suggests doing `http://::1:port`, which would be broken in the same way, as I would have expected `http://[::1]:port`.

Looking at the history, before July the document suggested to use `localhost`, but it was changed to the text we see "as it plays better with firewalls such as the default Windows 10 firewall".